_til/linux/install-ubuntu.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Install Ubuntu 
+category: linux
+---
+
+Do you want to move from Windows into a UNIX like dev environment? Ubuntu is an easy way to get your feet wet and try out Linux.
+
+## Creating bootable media
+Ubuntu comes from the web as a .iso file. Here's how to set it up on Windows using Rufus:
+* Download the Ubuntu image from [Canonical](https://www.ubuntu.com/download)
+* Download [Rufus](https://rufus.akeo.ie/)
+* Acquire a a USB flash drive of at least 2GB
+* Run Rufus and select your USB from the Devices dropdown
+* Click the CD ROM icon next to the FreeDOS menu entry and select your Ubuntu .iso
+* Click Start
+* Select yes if prompted to download/install Syslinux
+* Select ISO mode when prompted between ISO and DD modes
+* Confirm that the USB you selected is correct
+* Press OK and sit back!
+
+## Basic Installation
+
+Ubuntu ships with a very straight forward installer and makes set up a snap. Here are the only steps:
+* Choose a language and keyboard layout (Usually English-US for both)
+* Connect to a network (Optional, convenient to install updates during installation)
+* Choose to install updates
+* Choose to install 3rd party non-free items (Free as libre, not cost wise. The non-free items are non-open source, but will cost you nothing)
+* Create a user and hostname (hostname is the name of your computer)
+* Partition the disk (See below. Manual partitioning is generally not required as the installer will offer both dual-boot options and full replacement automatically)
```

Comment:
Change this to: Select how to install the OS (either overwrite the disk, install along side another OS or a more complicated layout (LINK here)) 

Prioritize the easy stuff, it makes things more accessible